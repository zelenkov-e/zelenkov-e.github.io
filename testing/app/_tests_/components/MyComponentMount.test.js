import React from 'react';
import { mount } from 'enzyme';
import { MyComponentMount } from './MyComponentMount';



describe("MyComponentMount", () => {
    it(" allows us to set props", () => {
      const wrapper = mount(<MyComponentMount name = "Zhenya" />);
    //   expect(wrapper.props().name).toEqual('Zhenya');
      expect(wrapper.props().name).toMatchSnapshot();
    //   wrapper.setProps({name:'Zhenya'});
    //   expect(wrapper.props().name).to.equal('Zhenya')
    });
  });

// 