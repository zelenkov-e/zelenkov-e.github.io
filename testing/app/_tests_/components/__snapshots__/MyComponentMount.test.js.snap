// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponentMount  allows us to set props 1`] = `"Zhenya"`;
